### DISCUSSION TITLE
mod_auth_openidc EL7 performance vs EL8/EL9

### DISCUSSION THREAD

**Discussion by jakke79** [OpenIDC/mod_auth_openidc#1139] [Q&A] [answered]
Hello! A while ago we noticed that mod_auth_openidc EL7 version was only available for the latest release. It was deleted from other releases? What will happen for EL7 in future? We tried to move on to EL9. Problem is that we are having some sort on performance issues with EL9 compared to EL7. Also error rates in our integration tests go up with EL9 version. We also noticed that EL7 package size twice as big compared to EL8/EL9, so is there some significant difference between EL7 vs EL8/EL9? Have you heard of any similiar issues?

**Comment by zandbelt** [comment, marked as answer]
Once EL7 becomes EOL mid-2024 the build of mod_auth_openidc for EL7 will only be offered under our commercial subscription support offerings. The performance on EL7 and EL8/9 is the same or better in our load tests - and we have not hear similar reports - which suggests that the difference must be in your application/environment,
